### DISCUSSION TITLE
Scheduler Reload

### DISCUSSION THREAD

**Discussion by ZacWarham** [nzbgetcom/nzbget#295] [Q&A] [answered]
I have an issue where the Unpacking (and sometimes the Checking) goes on forever (waited at least 12 hours) so to fix it I just go to the System settings and Reload.

Instead of having to monitor the Downloads, is there a way to use the Scheduler (or if not, an extension script) to run this periodically?

**Comment by dnzbk** [comment, marked as answer]
The scheduler doesn't support such a command, but this behavior can be achieved in an extension using the API method [reload](https://nzbget.com/documentation/api/reload/) via [XML/JSON-RPC](https://nzbget.com/documentation/api/). 

Concerning the problem itself, could you create an issue with more details on how to reproduce it? 
I'd look at it and try to fix it, thanks.

**Reply by ZacWarham** [reply]
> The scheduler doesn't support such a command, but this behavior can be achieved in an extension using the API method [reload](https://nzbget.com/documentation/api/reload/) via [XML/JSON-RPC](https://nzbget.com/documentation/api/).
> 
> Concerning the problem itself, could you create an issue with more details on how to reproduce it? I'd look at it and try to fix it, thanks.

I only just now saw your edit on this @dnzbk about trying to fix it. Re-creating consistently is problematic as it just occurs through general use on seemingly random files. I will create an issue describing my setup and linking to some other resources of people experiencing the same issue.
